lib/chart/barchart.ex

```diff
@@ -259,7 +259,13 @@ shown. You can force the range using `force_value_range/2`
 
     cat_band = OrdinalScale.get_band(category_scale, cat_data)
     bar_values = prepare_bar_values(series_values, value_scale, plot.type)
-    labels = Enum.map(series_values, fn val -> Scale.get_formatted_tick(value_scale, val) end)
+
+    labels =
```

Comment:
I think this is the wrong place - `Scale.get_formatted_tick` should return a string. I'll take a look. If we do it here we don't fix it in other places that use the scales.

Reply:
Ah, good point.